locales/nl/LC_MESSAGES/volto.po

```diff
@@ -30,16 +35,6 @@ msgstr "Omschrijving"
 msgid "Go to slide"
 msgstr ""
 
-#: components/schema
-# defaultMessage: Headtitle
-msgid "Headtitle"
```

Comment:
This is a breaking change, and we need to at least copy over the existing translations